vk-definitions.json

```diff
@@ -574,6 +574,15 @@
                 "GithubRepo": "prometheus_varnish_exporter",
                 "ReleaseName": "prometheus_varnish_exporter-{VERSION}.linux-amd64.tar.gz",
                 "Filename": "prometheus_varnish_exporter-{VERSION}.linux-amd64/prometheus_varnish_exporter"
+            },
+            {
+                "Cmd": "terrascan",
+                "VersionArg": "version",
+                "VersionRegexp": "version:\"v(.+?)\"",
```

Comment:
VersionRegexp is incorrect. This one works: **version: v([0-9]+.[0-9]+.[0-9]+)**

Reply:
corrected VersionRegexp